test/functions/cwailes/disambiguation/dbm_case_M.good

```diff
@@ -1 +1 @@
-foo1
+foo2
```

Comment:
This might be another interesting one to put in the "summary of changes" list.  My takeaway is that same-sized overloads are now preferred over same-signed overloads for params?

Reply:
It is already there 


``` chapel
dbm(42:int(64)); // now calls dbm(uint(64)) b/c of same-width rule
```

Reply:
Whoops, I had forgotten, thanks!